### PR TITLE
Take sourcetype filter for internal queries from inputs.conf

### PR DESCRIPTION
**Pull request by matiasmoreno876**
Hi team, 
this PR resolve issue [#1101](https://github.com/wazuh/wazuh-splunk/issues/1101)

Regards

### CODE REVIEW

File: SplunkAppForWazuh/appserver/static/js/controllers/settings/index/settingsIndexCtrl.js [hunk 1]
```diff
@@ -20,41 +20,83 @@ define([
       this.currentDataService = $currentDataService
       this.urlTokenModel = $urlTokenModel
       this.notification = $notificationService
+      this.selectedIndex = 'wazuh'
 
       this.dropdown = new Dropdown(
         'inputIndexes',
-        `| metasearch index=* sourcetype=wazuh | stats count by index, sourcetype | fields index`,
+        `| metasearch index=* sourcetype=*wazuh* | stats count by index, sourcetype | fields index`,
```
Comment (Desvelao) [inline SplunkAppForWazuh/appserver/static/js/controllers/settings/index/settingsIndexCtrl.js:27]: **nitpick**: The reference to `wazuh` should be variable?

Comment (frankeros) [reply SplunkAppForWazuh/appserver/static/js/controllers/settings/index/settingsIndexCtrl.js:27]: I think it's not necessary for this, it's just the minimum requirement to consider an index as valid.

File: SplunkAppForWazuh/appserver/static/js/services/sourceTypeStorageService/sourceTypeStorageService.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            this.sessionStorage.selectedSourceType = `{"sourceType":"${sourceType}"}`
+        }
+
+        /**
+         * Returns currently selected SourceType
+         * @param {String} sourceType
+         */
+        getSourceType() {
+            if (this.sessionStorage.selectedSourceType) {
+                return JSON.parse(this.sessionStorage.selectedSourceType)
```
Comment (asteriscos) [inline SplunkAppForWazuh/appserver/static/js/services/sourceTypeStorageService/sourceTypeStorageService.js:30]: is it possible that the JSON.parse fails? should a possible exception be handled?

**Review by asteriscos — COMMENTED** [review]
Possible improvement.

File: SplunkAppForWazuh/appserver/static/js/services/sourceTypeStorageService/sourceTypeStorageService.js (hunk 2 above)
Comment (matiasmoreno876) [reply SplunkAppForWazuh/appserver/static/js/services/sourceTypeStorageService/sourceTypeStorageService.js:30]: Yes, as every expression is possible to fail, the use of try and catch seems correct to me. Thanks a lot for the suggestion.

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/docker/agentsDockerCtrl.js [hunk 3]
```diff
@@ -78,45 +78,45 @@ define([
       this.filters = this.getFilters()
 
       this.vizz = [
-        //`${this.filters} sourcetype=wazuh | timechart span=1h count`,
+        //`${this.filters} sourcetype=${this.currentDataService.getSourceType().sourceType} | timechart span=1h count`,
```
Comment (gabiwassan) [inline SplunkAppForWazuh/appserver/static/js/controllers/agents/docker/agentsDockerCtrl.js:81]: Clear this commented code 🤓

**Review by gabiwassan — COMMENTED** [review]
LTGM

**Review by gabiwassan — APPROVED** [review]

**Pull request merged**